index.html

```diff
@@ -155,6 +155,11 @@ <h2 class="fw-bold">Take a look inside</h2>
             y.parentNode.insertBefore(t, y);
           })(window, document, "clarity", "script", "4q1wajdktz");
         </script>
+        <script>
+          if (window.navigator.language.slice(0, 2) == 'ja'){
+            location.href = 'ja-jp/index.html';
+          }
+        </script>
```

Comment:
If someone with locale japan want's to view the en file, you immediately send them back. How about showing a small toast at the bottom suggesting to switch to ja?

Reply:
Additional option: landing page is english, also introduce en-us/index.html which is an exact copy that can be used for language switching?

Reply:
> If someone with locale japan want's to view the en file, you immediately send them back. How about showing a small toast at the bottom suggesting to switch to ja?
I think this is a great idea!
Could you make a toast design for that?